### PR TITLE
ICU-22100 Remove unicode blocks from Japanese ML phrase breaking

### PR DESCRIPTION
**Pull request by allensu05**
Unicode code block is a less significant feature in breakpoint evaluation. The performance can be improved and quality has no impact when using the re-trained model which removes the unicode block.

[Performance evaluation](https://docs.google.com/document/d/1zdvdoSB-EAiDacYX1SZDe_HRXL6r1OHcTGALiIQErUA/edit#heading=h.71hhj0cduyur)
[Model footprint](https://docs.google.com/document/d/1zdvdoSB-EAiDacYX1SZDe_HRXL6r1OHcTGALiIQErUA/edit#heading=h.u4udvhv6vo4o)
The size of the model file (jaml.res) reduces from 9880 byes to 6820 bytes.

##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-22100
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [x] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

**Comment by FrankYFTang** [comment]
could. you change the first comment to explain what this PR is intended to do? I cannot tell from https://unicode-org.atlassian.net/browse/ICU-22100 That is a meta bug for the whole feature and I cannot tell why you need this PR and what you are trying to do here.

**Comment by allensu05** [comment]
> could. you change the first comment to explain what this PR is intended to do? I cannot tell from https://unicode-org.atlassian.net/browse/ICU-22100 That is a meta bug for the whole feature and I cannot tell why you need this PR and what you are trying to do here.

Done. I add more description in the title.

**Comment by FrankYFTang** [comment]
Mostly code removeal. Looks reasonable to me. Could you summarize the size change of the dat file before/after this PR? I assume it reduce the size of dat file.

**Comment by allensu05** [comment]
> Mostly code removeal. Looks reasonable to me. Could you summarize the size change of the dat file before/after this PR? I assume it reduce the size of dat file.

Done. Add in the title description.

**Review by FrankYFTang — DISMISSED** [review]

**Comment by FrankYFTang** [comment]
Please squash to a single commit and I will close and merge it.

**Review by FrankYFTang — APPROVED** [review]

**Comment by markusicu** [comment]
nice, thanks!
